### PR TITLE
Adding Consul

### PR DESCRIPTION
**Pull request by jonatan-ivanov**
Consul is a distributed, highly available, datacenter aware registry/discovery service with key-value store and failure detection developed by HashiCorp. https://www.consul.io/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -522,6 +522,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools for managing microservices, i.e. creation, coordination or discovery.*
 
 * [Apollo](https://spotify.github.io/apollo/) -  Libraries for writing composable microservices.
+* [Consul](https://www.consul.io/) -  Distributed, highly available and datacenter-aware registry/discovery service.
```
Comment (akullpp) [inline README.md]: Same whitespace issue as in #429

Comment (jonatan-ivanov) [inline README.md]: Thanks, done

**Comment by jbduncan** [comment]
It's not clear to me why this PR was accepted. AFAICT, Consul is written in Go, not Java.

@akullpp, @jonatan-ivanov, can you explain to me why this PR was accepted and what benefit it brings to Java? :)

**Comment by akullpp** [comment]
@jbduncan see https://github.com/akullpp/awesome-java/commit/026e5f5e70a1f4b087c1c50451def3f02789dc84

**Comment by jbduncan** [comment]
Great, thanks. :)

**Pull request merged**